packages/analytics-js-integrations/src/integrations/GA4/browser.js

```diff
@@ -128,14 +129,11 @@ export default class GA4 {
       this.sessionNumber = sessionNumber;
     });
 
-    ScriptLoader(
-      'google-analytics 4',
-      `https://www.googletagmanager.com/gtag/js?id=${measurementId}`,
-    );
+    ScriptLoader('google-analytics 4', `${sdkBaseUrl}/gtag/js?id=${measurementId}`);
```

Comment:
Let's handle the case where there is a trailing `/` in the base URL.

Reply:
Done. Check line 40.